dev/MRTCore/mrt/Microsoft.WindowsApp.Resources/projection/Microsoft.WindowsApp.Resources.Projection.csproj

```diff
@@ -2,7 +2,7 @@
   
   <PropertyGroup>
     <TargetFramework>net5.0-windows10.0.18362.0</TargetFramework>
```

Comment:
since you are here, can you change it to 17763?

Reply:
and line 20.

Reply:
I thought we didn't need to change that cuz it would default to the min version :/